src/data/real/pi.lean

```diff
@@ -142,4 +142,109 @@ lemma pi_lt_3141593 : pi < 3.141593 := by pi_upper_bound [
   27720/19601, 56935/30813, 49359/25163, 258754/130003, 113599/56868, 1101994/551163,
   8671537/4336095, 3877807/1938940, 52483813/26242030, 56946167/28473117, 23798415/11899211]
 
+
+/-! ### Leibniz's Series for Pi -/
+
+noncomputable theory
+open real filter set
+open_locale classical big_operators topological_space
+local notation `|`x`|` := abs x
+local notation `π` := real.pi
+
```

Comment:
Could you explain in a comment the proof that you are going to formalize below, using the same notation as in the formal proof and maybe using some step numbers that you could also refer to in the formal proof?